packages/api/src/user.js

```diff
@@ -172,3 +177,28 @@ export async function userTokensPost (request, env) {
 
   return new JSONResponse()
 }
+
+/**
+ * Retrieve user auth tokens.
+ *
+ * @param {AuthenticatedRequest} request
+ * @param {import('./env').Env} env
+ */
+export async function userTokensGet (request, env) {
+  const res = await env.db.query(gql`
+    mutation FindAuthTokensByUser($user: ID!) {
+      # Paginated but users are probably not going to have tons of these.
+      # Note: 100,000 is the max page size.
```

Comment:
Should we just put a cap on it? Let's prevent users making more than 100,000 tokens. They can get in touch for the enterprise license if they need more.

Reply:
The default page size is 64. I can remove the size param to get that many...

I'll open an issue to prevent token creation abuse 😄